src/components/GDPRConsent.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React, { useEffect } from 'react';
+
+import { GTM_ID } from '../constants';
+import CookieConsent from "react-cookie-consent";
+import TagManager from 'react-gtm-module';
+
+
+const GDPRConsent = () => {
+
+  useEffect(() => {
+    // Just to ensure that when we initialize GTM, the message on this component will be shown
+    TagManager.initialize({ gtmId: GTM_ID });
```

Comment:
That's not the best place to do it, right? I think we can move forward but let's open an issue to get it fixed ASAP.

Reply:
Yes! I'm already talinkg about it here: https://github.com/HathorNetwork/internal-issues/issues/29#issuecomment-867080613